lego/apps/reactions/serializers.py

```diff
@@ -1,31 +1,43 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db import IntegrityError
 from rest_framework import serializers
-
-from lego.apps.reactions.models import Reaction, ReactionType
+from rest_framework.exceptions import ValidationError
+
+from lego.apps.reactions.exceptions import (
+    APIReactionExists,
+    APITooManyReactions,
+    ReactionExists,
+    TooManyReactions,
+)
+from lego.apps.reactions.models import Reaction
 from lego.apps.users.serializers.users import PublicUserSerializer
 from lego.utils.serializers import BasisModelSerializer, GenericRelationField
 
 
-class ReactionTypeSerializer(BasisModelSerializer):
-    class Meta:
-        model = ReactionType
-        fields = ("short_code", "unicode")
-
-
 class GroupedReactionSerializer(serializers.Serializer):
```

Comment:
Is this serializer used?

Reply:
It is not currently, the plan was to use it once the logic to get the grouped reactions has been moved to sql. I'll remove the serializer now.